CompositeUi/src/scss/components/_notification.scss

```diff
@@ -53,5 +53,5 @@ $notification-height: 50px !default;
   height: 15px;
   margin-left: 10px;
   margin-right: 10px;
-  width: 15px
+  width: 15px;
```

Comment:
This bug is already fixed in the master branch. You need to rebase master before merge this PR.